roles/openshift_control_plane/tasks/static_shim.yml

```diff
@@ -1,17 +1,29 @@
 ---
 # TODO: package this?
-- name: Copy static master scripts
+- name: Copy static master exec scripts
   copy:
-    src: "{{ item }}"
+    src: "scripts/{{ l_runtime }}/master-exec"
```

Comment:
this still needs a crio scipt right? (which uses crictl)

Reply:
Yes, added a TODO